includes/languages/english/account.php

```diff
@@ -17,7 +17,6 @@
 define('TABLE_HEADING_ORDER_NUMBER', 'No.');
 define('TABLE_HEADING_SHIPPED_TO', 'Ship To');
 define('TABLE_HEADING_STATUS', 'Status');
-define('TABLE_HEADING_TOTAL', 'Total');

```

Comment:
Was this intentional

Reply:
Yes, same issue with that define.  But I can break it out if you prefer.